packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -4017,15 +4281,15 @@ enum DebugSemanticsDumpOrder {
   traversalOrder,
 }
 
-String _concatStrings({
-  required String thisString,
-  required String otherString,
+AttributedString _concatAttributedString({
+  required AttributedString thisAttributedString,
```

Comment:
Wondering whether AttributedString could just implement `operator +` for concatenation.